src/test/java/com/nextcloud/android/sso/api/TestRetrofitAPI.java

```diff
@@ -438,14 +439,14 @@ public void testQueryParamInUrl() {
             fail(e.getMessage());
         }
 
-        Map<String, String> map = new HashMap<>();
-        map.put("format", "json");
-        map.put("test", "1");
+        List<Pair<String, String>> params = new ArrayList<>();
+        params.add(new Pair<>("format", "json"));
+        params.add(new Pair<>("test", "1"));
```

Comment:
Can we extend this test, so that we see that with doubled names, old parameter only uses one, but new system uses both?

Reply:
> Looks good so far.
> Can you also add this to readme?

@tobiasKaminsky Can you please explain which documentation you are talking about? I could help.